server/postgres/migrations/000000_initial.sql

```diff
@@ -72,7 +72,6 @@ CREATE TABLE users(
     is_owner BOOLEAN DEFAULT FALSE, -- has the ability to start conversations
     zinvite VARCHAR(300), -- The initial zinvite used to create the user, can be used for attribution (may be null)
     oinvite VARCHAR(300), -- The oinvite used to create the user, or to upgrade the user to a conversation owner.
-    plan SMALLINT DEFAULT 0,
```

Comment:
@ballPointPenguin I think we should removed this from initial, since it might cause an error if you try to remove the field when it doesn't exist.

That having been said, maybe it's worth maintaining both an `initial` and a `current` schema, so that you could just start from current without creating and dropping a bunch of tables/fields unnecessarily? Maybe not worth the maintenance though.

Reply:
Nevermind; Realize we are using `DROP IF EXISTS` in the migrations below; Ignore me :-)